rules/forbidden_actions.tex

```diff
@@ -103,24 +103,24 @@ \subsubsection{Penalty Area During Penalty Kick}
 If a robot enters the relevant penalty area during a penalty kick, except for the goalkeeper (defending team) and one robot of the attacking team, ``Illegal Position'' is called.
 
 \subsection{Forbidden Motion}
-Robots that begin moving their legs or locomote in any fashion before reaching the specified state in the subsections
-will be penalized \textit{in place} on the field for \qty{15}{\second} in \textit{playing}. It's important to note that the penalty duration only counts down during the playing state.
+Robots that begin moving their legs or locomote in any fashion before reaching the specified state in the following subsections
+will be penalized \textit{in place} on the field.
 ``Forbidden Motion'' penalties do not follow the standard removal procedure, and hence do not count towards the incremental penalty count.
-A robot will be moved back to its original position if it has moved significantly before becoming penalized.
+A robot will be moved back to its original position if it has moved significantly before being penalized.
 
 \subsubsection{Motion in Standby}
 \label{sec:motion_in_standby}
 
-Robots that begin to move during \texttt{standby} (\ie before the head referee does the visual signal).
-The head referee will call ``Forbidden Motion in Standby\textless robot\textgreater''.
-Note that responding to any false signal of any fashion will result in this penalty.
+If a robot begins to move during \texttt{standby} (\ie before the head referee does the visual signal), all robots from that team will be penalized until the end of the \textit{ready} state.
```

Comment:
We should keep this in mind, since the GameController has to be updated so that this penalty is directly given to all robots of the same team.

Reply:
This is already clearly defined in the explanation of the standby state, but I will update here to better reflect that.